primefaces/src/main/java/org/primefaces/component/column/ColumnBase.java

```diff
@@ -79,6 +83,8 @@ public enum PropertyKeys {
         width
     }
 
+    private Converter<?> converter;
```

Comment:
im not sure if this caching works correctly with p:columns and converter=#{myModel.myConverterId} ?

Reply:
I was struggling with the converter attribute to be honest. It now takes both a converter instance and a converterId... it all feels a bit like a hack TBO.

Regarding caching, I'll have a second look at it. Needs to be something like

https://github.com/apache/myfaces/blob/master/api/src/main/java/jakarta/faces/component/UIOutput.java
https://github.com/eclipse-ee4j/mojarra/blob/master/impl/src/main/java/jakarta/faces/component/UIOutput.java

Reply:
I would not cache it for now and use stuff from ComponentUtils or introduce a new method there
And add to the Docs to use AppScoped converters